helpers/handlebars/helpers/index.js

```diff
@@ -127,6 +128,12 @@ module.exports = {
 		}
 		return options.fn(this);
 	},
+	global: (globalVariable, options) => {
+		if (global[globalVariable] !== undefined) {
```

Comment:
this may fail, better use something like obj.hasProperty or in operator instead, see
https://stackoverflow.com/a/1894803

Reply:
add globals to root https://handlebarsjs.com/api-reference/data-variables.html#root instead